src/Tribe/Views/V2/Assets.php

```diff
@@ -693,20 +695,27 @@ public function should_enqueue_single_event_block_editor_styles() {
 
 		// Bail if not Single Event V2.
 		if ( ! tribe_events_single_view_v2_is_enabled() ) {
-			return false;
+			$should_enqueue = false;
```

Comment:
Do we need this one any more? It returns true unless someone is using a deprecated filter.
https://github.com/the-events-calendar/the-events-calendar/blob/c582758f351ce32132d1d73cb25a8f5a443c418c/src/functions/views/provider.php#L182